services/slam/builtin/builtin_test.go

```diff
@@ -781,11 +781,11 @@ func TestORBSLAMDataProcess(t *testing.T) {
 
 		n := 5
 		// Note: timePadding is required to allow the sub processes to be fully completed during test
-		time.Sleep(time.Millisecond * time.Duration((n)*(validDataRateMS+timePadding*2)))
+		time.Sleep(time.Millisecond * time.Duration((n)*(validDataRateMS+timePadding)))
```

Comment:
Sleeping like this is unreliable in the face of parallelized CI. We should figure out a way to sleep less and signal more.

Reply:
There's also https://pkg.go.dev/go.viam.com/utils@v0.0.5/testutils#WaitForAssertion:
```
	testutils.WaitForAssertion(t, func(tb testing.TB) {
		tb.Helper()
		test.That(tb, len(robotClient.ResourceNames()), test.ShouldEqual, 3)
	})
```

Reply:
That will just wait forever basically which is nice sometimes

Reply:
Was this in reply to my most recent commit? https://github.com/viamrobotics/rdk/pull/1401/commits/663a505c26a44ef6b48a2d5f9e0df758caf0aefd